src/equisolve/numpy/models/linear_model.py

```diff
@@ -47,20 +47,28 @@ class Ridge:
 
     def __init__(
         self,
-        parameter_keys: Union[List[str], str],
-        alpha: TensorMap,
+        parameter_keys: Union[List[str], str] = None,
+        alpha: TensorMap = None,
         rcond: float = 1e-13,
     ) -> None:
+        if parameter_keys is None:
+            paramater_keys = ["values", "positions"]
         if type(parameter_keys) not in (list, tuple, np.ndarray):
             self.parameter_keys = [parameter_keys]
         else:
             self.parameter_keys = parameter_keys
 
+
+        # TODO(philip) can we make a good default alpha parameter out of paramater_keys?
+        if alpha is None:
+            raise NotImplemented("Ridge still needs a good default alpha value")
```

Comment:
The easiest would be to choose 1 in the fit method if this is `None`. But we can also leave this as a required argument at let the user think about it.